server/methods/matching_engine.py

```
@@ -109,21 +109,15 @@ def analyser_match(self, user: User, job: Offres_FT) -> dict:
-            
-            # Nettoyage Markdown éventuel
-            if content.startswith("```json"):
-                content = content.replace("```json", "").replace("```", "")
-            elif content.startswith("```"):
-                content = content.replace("```", "")
+            if content.startswith("```json"): content = content.replace("```json", "").replace("```", "")
+            elif content.startswith("```"): content = content.replace("```", "")
             
             return json.loads(content)
 
         except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)